### ISSUE TITLE
summarizing the model

### ISSUE THREAD

**Issue by lmeeden1** [Calysto/conx#21]
Include the number of parameters, and get the actual activation function from keras. Also when there is no dropout, don't report anything about dropout.

**Comment by dsblank** [comment]
Every conx layer now has a pointer to its associated keras layer:

```
net["input"].keras_layer
```
There are useful things there, such as incoming_connections, outgoing_connections, and activation.

**Comment by dsblank** [comment]
Maybe just use net.model.summary() instead?
